src/tests/nativeaot/SmokeTests/MultiModule/MultiModule.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <CLRTestKind>BuildAndRun</CLRTestKind>
+    <CLRTestPriority>0</CLRTestPriority>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+
+    <!-- This test always runs as multimodule. It's not supported if we don't have framework object files. -->
+    <CLRTestTargetUnsupported Condition="'$(BuildNativeAotFrameworkObjects)' != 'true'">true</CLRTestTargetUnsupported>
+    <IlcMultiModule>true</IlcMultiModule>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="MultiModule.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="Library.csproj" />
+  </ItemGroup>
+
+  <!--
+    When building a library as a reference, the returned build outputs include both the 
+    IL assembly produced by CSC and the native obj produced by Ilc. Separate the native
+    object file here so it isn't passed to CSC, and instead redirect it to be picked up
+    by the LinkNative target.
+  -->
+  <Target Name="RemoveObjFiles" AfterTargets="ResolveProjectReferences">
```

Comment:
Non-blocking - I wonder whether this logic should be rather put in one of the common scripts like Directory.Build.targets so that it doesn't need to be open-coded in the individual tests. Or is this special to just this one test?

Reply:
It's specific to the MultiModule compilation mode that we don't ship... it should eventually be part of the product. We don't need it for other tests.